<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sozlesme</class>
 <widget class="QWidget" name="Sozlesme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1818</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>431</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
</string>
   </property>
   <property name="title">
    <string>Yeni Randevu</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
</string>
    </property>
    <property name="title">
     <string>                            Araç Bilgileri</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_Plaka_Getir">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 8pt &quot;Gill Sans Ultra Bold&quot;;</string>
     </property>
     <property name="text">
      <string>Plaka</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_sonKm">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Son KM</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
</string>
    </property>
    <property name="title">
     <string>                              Kira Bilgileri</string>
    </property>
    <widget class="QDateTimeEdit" name="dateTimeEdit_AlisTarih">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_TeslimTarih">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Alış Tarihi</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Teslim Tarihi</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ToplamTutar">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Toplam Tutar (TL)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
</string>
    </property>
    <property name="title">
     <string>                              Müşteri Bilgileri</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Gill Sans Ultra Bold&quot;;</string>
     </property>
     <property name="text">
      <string>T.C No</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_TCNo_Getir">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_isim">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Soyisim">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Telefon">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>İsim</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Soyisim</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Telefon</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>1381</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
</string>
   </property>
   <property name="title">
    <string>Randevular</string>
   </property>
   <widget class="QTableView" name="tableView_AracListesi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1341</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Kirala">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>630</y>
      <width>196</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    background-color: navy;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
    min-width: 10em;
    padding: 6px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Kirala</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exceleAktar">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>630</y>
      <width>196</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    background-color: navy;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 15px;
    min-width: 10em;
    padding: 6px;
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Excel'e Aktar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
